sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClient.java

```diff
@@ -230,30 +231,36 @@ private CommunicationUserIdentifierAndToken userWithAccessTokenResultConverter(
     /**
      * Exchanges an AAD access token of a Teams User for a new Communication Identity access token.
      *
-     * @param teamsUserAadToken AAD access token of a Teams User to acquire Communication Identity access token.
+     * @param options {@link GetTokenForTeamsUserOptions} request options used to exchange an AAD access token of a Teams User for a new Communication Identity access token.
      * @return Communication Identity access token.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public AccessToken getTokenForTeamsUser(String teamsUserAadToken) {
-        TeamsUserAccessTokenRequest requestBody = new TeamsUserAccessTokenRequest();
-        requestBody.setToken(teamsUserAadToken);
+    public AccessToken getTokenForTeamsUser(GetTokenForTeamsUserOptions options) {
+        TeamsUserExchangeTokenRequest requestBody = createTeamsUserExchangeTokenRequest(options);
         CommunicationIdentityAccessToken rawToken = client.exchangeTeamsUserAccessToken(requestBody);
         return new AccessToken(rawToken.getToken(), rawToken.getExpiresOn());
     }
 
+    private TeamsUserExchangeTokenRequest createTeamsUserExchangeTokenRequest(GetTokenForTeamsUserOptions options) {
```

Comment:
Same code as in async version of this class. Maybe it can be extracted and reused.